test/util/report.js

```diff
@@ -260,14 +260,15 @@ fs.readdirSync(testsPath).forEach(function (testFile) {
   }
 });
 
-console.log('\nExamples without any regression tests:\n');
+console.log('\n## Regression test coverage:\n');
+console.log('\n### Examples without any regression tests:\n');
 console.log(examplesWithNoTestsReport || 'None found.\n');
-console.log('\nExamples missing some regression tests:\n');
+console.log('\n### Examples missing some regression tests:\n');
 console.log(examplesMissingSomeTestsReport || 'None found.\n');
-console.log('\nExamples documentation table rows without data-test-ids:\n');
+console.log('\n### Examples documentation table rows without data-test-ids:\n');
 console.log(missingTestIdsReport || 'None found.\n');
 
-console.log('SUMMARTY:\n');
+console.log('### SUMMARTY:\n');
 console.log('  ' + exampleFiles.length + ' example pages found.');
```

Comment:
The whitspace padding gets lost, so it can probably get dropped here and the lines below. Not sure if `\t` would work instead if the indent should be kept

Reply:
That white space padding is not super important, it was just ot make the console output more readable and less of a wall of text. In this context I think it's totally fine!

Reply:
Random though, you could just make these 2 items a list